api/src/uvisor-input.S

```diff
@@ -93,14 +91,23 @@ uvisor_config:
     /* Privileged system IRQ hooks */
     .long __uvisor_priv_sys_irq_hooks
 
+/* uVisor mode of operation
+ * 0: Disabled
+ * 1: Permissive
+ * 2: Enabled */
```

Comment:
Yeah unfortunately the names would be UVISOR_ENABLED, _DISABLED, _PERMISSIVE and they're not an enum, but just preprocessor definitions. It would be equally outdated in case of change. I'd rather remove the comment altogether, what do you think?